packages/editor/src/components/editor-help/view-sections.native.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * External dependencies
+ */
+import { Text, Image } from 'react-native';
+
+/**
+ * WordPress dependencies
+ */
+import { useMemo } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import styles from './style.scss';
+
+export const HelpDetailBodyText = ( { text } ) => {
+	return <Text style={ styles.helpDetailBody }>{ text }</Text>;
+};
+
+export const HelpDetailSectionHeadingText = ( { text } ) => {
+	return <Text style={ styles.helpDetailSectionHeading }>{ text }</Text>;
+};
+
+export const HelpDetailImage = ( { source } ) => {
+	return useMemo(
+		() => <Image source={ source } style={ styles.helpDetailImage } />,
+		[ source ]
+	);
```

Comment:
Is the use of `useMemo` an attempt to avoid the `BottomSheet.SubSheet` flickering mentioned in the PR description? Should we retain this memoization now that `BottomSheet.SubSheet` is no longer used?

Reply:
That's a great question. If I remove this code, then the flickering does not happen but I left it seemed like a good optimization since scrolling was causing an almost infinite reload of those image. From a performance standpoint however, I do not see any noticeable difference when interacting with the bottom sheet so I've removed it. It can be added back in the future if deemed necessary. Thanks David! Updated in b68013cd2d5c44ae550a649d54ecc2a47250e38f